src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -842,7 +842,7 @@ namespace IceThermalStorage {
             // Convert GJ to J by multiplying by 10^9
             // Convert J to W-hr by dividing by number of seconds in an hour (3600)
             state.dataIceThermalStorage->DetailedIceStorage(iceNum).NomCapacity =
-                state.dataIPShortCut->rNumericArgs(1) * (1.e+09) / (DataGlobalConstants::SecInHour);
+                state.dataIPShortCut->rNumericArgs(1) * (1.e+09) / (Constant::SecInHour);
```

Comment:
In PackagedThermalStorageCoil:

    constexpr Real64 gigaJoulesToJoules = 1.e+09;